config/initializers/date_picker.rb

```diff
@@ -0,0 +1 @@
+require 'bootstrap-datepicker-rails'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Final newline missing.